package.json

```diff
@@ -27,8 +27,9 @@
     "downshift": "^5.0.5",
     "esm": "^3.2.25",
     "firebase": "^7.15.4",
-    "googleapis": "^39.2.0",
```

Comment:
I looked it up - it's a dependency because our production system depends on this package to work [npm: Specifying dependencies and devDependencies in a package.json file](https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file).

The `build` script won't include this in the front-end bundle, so I think we are good. If you feel strongly I can move it to dev-dependency, but I don't think it does any harm.

Reply:
Don't feel strongly. 😁